mempool/reactor.go

```diff
@@ -168,6 +164,27 @@ func (memR *Reactor) Receive(e p2p.Envelope) {
 	// broadcasting happens from go routines per peer
 }
 
+// TryAddTx attempts to add an incoming transaction to the mempool.
+// When the sender is nil, it means the transaction comes from an RPC endpoint.
+func (memR *Reactor) TryAddTx(tx types.Tx, sender p2p.Peer) (*abcicli.ReqRes, error) {
+	senderID := noSender
+	if sender != nil {
+		senderID = sender.ID()
+	}
+
+	reqRes, err := memR.mempool.CheckTx(tx, senderID)
+	switch {
+	case errors.Is(err, ErrTxInCache):
+		memR.Logger.Debug("Tx already exists in cache", "tx", log.NewLazySprintf("%v", tx.Hash()))
```

Comment:
```suggestion
		memR.Logger.Debug("tx already exists in cache", "tx", log.NewLazySprintf("%v", tx.Hash()))
```

Reply:
why? https://github.com/cometbft/cometbft/blob/a3ec97e3b15afafd57b44305d4cc40a801d4bc35/CONTRIBUTING.md#log-messages Unlike Go errors, log messages must have the first letter of the message capitalized.

Reply:
I'll leave it as it is.